README.md

```diff
@@ -6,13 +6,7 @@ Martha
 Google Cloud Functions for resolving [DOS](https://data-object-service.readthedocs.io/en/latest/) URIs.
 
 # Martha v1
-To call `martha_v1`, perform an HTTP `POST` to the appropriate URL. The body of the request must be a JSON Object with
-two values: a [DOS](https://data-object-service.readthedocs.io/en/latest/) URL and a regex pattern. Martha uses the
-DOS URL to retrieve a data object, unpacks it, and returns the first link that matches the specified regex pattern.
-For more details look [here](https://docs.google.com/document/d/1AyyI6L43te_DFWh8dXAiX0Qx-8f3JLKUIZe6xFwKMb0/edit#).
-
-Staging: https://us-central1-broad-dsde-staging.cloudfunctions.net/martha_v1
-Production: https://us-central1-broad-dsde-prod.cloudfunctions.net/martha_v1
+Deprecated as of March 2020. Please use Martha v2.
```

Comment:
Deprecation usually implies that functionality still works. In this case, is the function being left behind, or removed from GCF?

Reply:
No. We are deleting the function. Should this message be updated to say `Deleted as of ...`?

Reply:
Yep, sounds good to me